src/main.ts

```diff
@@ -10,9 +10,16 @@ import { selectOs } from "./commands/select-os";
 dotenv.config();
 
 const vorpal = new Vorpal();
-
-PathUtils.checkExists(PathUtils.savePath());
-PathUtils.checkExists(PathUtils.projectPath());
+PathUtils.savePath().then((path) => {
+  PathUtils.checkExists(path);
+}).catch((err) => {
+  throw new Error("Error when finding default paths: " + err);
+});
+PathUtils.projectPath().then((path) => {
+  PathUtils.checkExists(path);
+}).catch((err) => {
+  throw new Error("Error when finding default paths: " + err);
+});
```

Comment:
use callbacks instead of promises

Reply:
why?

Reply:
Certain PathUtils functions have been converted to asynchronous, when attempting to await functions an error emerges as follows:
Top-level 'await' expressions are only allowed when the 'module' option is set to 'esnext' or 'system', and the 'target' option is set to 'es2017' or higher.ts(1378)

Reply:
wrap the main.ts inside 
() async => {
}()